### DISCUSSION TITLE
Bot não está autenticando

### DISCUSSION THREAD

**Discussion by HelioSilva** [HelioSilva/Whatsapp-bot-NodeJS#2] [Q&A] [answered]
Se você está com problema de autenticação, verifique se está usando o pacote Venom-BOT. Na gravação do vídeo o pacote Sulla estava funcionando mas estava prestes a ser descontinuado, por isso tive que fazer a alteração para o Venom que continuou com o projeto.

**Comment by guiprj** [comment, marked as answer]
Olá Helio!

Primeiramente parabéns pelo trabalho!
Estou tentando fazer o deploy para o heroku, mas não estou conseguindo, ele dá um erro H10, e não está inicializando com a sessão que criei. Poderia me ajudar?

Obrigado!

**Comment by HelioSilva** [comment]
Olá @guiprj , 

Obrigado!  Vou fazer um teste aqui também, as vezes tem alteração no repo do venom e acontece esses problemas de inicialização.

**Reply by guiprj** [reply]
Ok!

Fico no aguardo, Obrigado Helio!
